docs/setuptools.txt

```diff
@@ -2293,6 +2293,44 @@ Metadata and options are set in the config sections of the same name.
 
 * Unknown keys are ignored.
 
+setup.cfg-only projects
+=======================
+
+.. note:: New in 40.9.0 (03 Apr 2019).
+
+Latest versions of ``setuptools`` emulate a dummy ``setup.py`` module
```

Comment:
It should be "*The* latest versions of..", but I also think there's no need to qualify it as "The latest versions" - this documentation applies to the latest version, and the "version added" note should suffice to explain when the behavior changed. At some point this behavior will be very old and it won't matter so much that you need a "recent" version.

Reply:
Yeah... I was thinking about this sentence as the section intro. That's why I've started it like this. versionadded sounds good